dali/operators/generic/reduce/reduce.h

```diff
@@ -28,16 +28,53 @@
 #define REDUCE_TYPES (uint8_t, int8_t, uint16_t, int16_t, uint32_t, int32_t, uint64_t, int64_t, float)  // NOLINT
 
 namespace dali {
+namespace detail {
+
+class AxesHelper {
+ public:
+  explicit inline AxesHelper(const OpSpec &spec) {
+    const bool has_axes_arg = spec.HasArgument("axes");
+    const bool has_axis_names_arg = spec.HasArgument("axes_names");
```

Comment:
```suggestion
    const bool has_axis_names_arg = spec.HasArgument("axis_names");
```
We call it like that in other operators

Reply:
Done